scanners/xamarin/xamarin.go

```diff
@@ -39,7 +39,7 @@ const (
 	xamarinPlatformInputKey     = "xamarin_platform"
 	xamarinPlatformInputEnvKey  = "BITRISE_XAMARIN_PLATFORM"
 	xamarinPlatformInputTitle   = "Xamarin solution platform"
-	xamarinPlatformInputSummary = ""
+	xamarinPlatformInputSummary = "The Xamarin solution configuration that you wish to run in your first build. You can change this at any time in your Workflows."
```

Comment:
You have switched the `xamarinPlatformInputSummary` and the `xamarinConfigurationInputSummary`.

Well, the xamarinConfigurationInputSummary is empty currently. :)